### ISSUE TITLE
namespace for migrations

### ISSUE THREAD

**Issue by lorinczelod** [yiisoft/yii2#9469]
It would be useful to add a namespace to migrations. For example:
I want to apply the `vendor/yiisoft/yii2/rbac/migrations/m140506_102106_rbac_init.php` migration.
For that now I need to run: ``yii migrate --migrationPath=@yii/rbac/migrations``
But would it be more convenient, if I made a local 'reference':
My migration in `console/migrations/m150819_075235_rbac_init.php` would be:
``class m150819_075235_rbac_init extends \yii\rbac\migrations\m140506_102106_rbac_init`` { }
In this case the simple `yii migration` would install that 'custom' migration, too.

**Comment by lorinczelod** [comment]
now I am using this working method to workaround the problem:
```
<?php
$alias = Yii::getAlias('@yii/rbac/migrations/m140506_102106_rbac_init.php');
Yii::$classMap['m140506_102106_rbac_init'] = $alias;

class m150819_075235_rbac_init extends m140506_102106_rbac_init { }
```
PS. my issue title is probably wrong, as it gives reference to the resolve, not to the problem. Please feel free to change.

**Comment by klimov-paul** [comment]
Why you don't use old good `require_once` ?

**Comment by lorinczelod** [comment]
[klimov-paul](https://github.com/klimov-paul), you have right!

**Comment by klimov-paul** [comment]
Resolved by commit 8aa0e85
